_data/simple-icons.json

```
@@ -2185,6 +2185,11 @@
             "hex": "D24939",
             "source": "https://wiki.jenkins-ci.org/display/JENKINS/Logo"
         },
+        {
+            "title": "Jenkins X",
+            "hex": "73C3D5",
+            "source": "https://github.com/cdfoundation/artwork/blob/master/jenkinsx/icon/black/jenkinsx-icon-black.svg"
```

I would prefer the source URL to be updated to either `https://github.com/cdfoundation/artwork` as that is more likely to persist than the specific file (i.e. `https://github.com/cdfoundation/artwork/blob/master/jenkinsx/icon/black/jenkinsx-icon-black.svg`).

```suggestion
            "source": "https://github.com/cdfoundation/artwork"
```

It will be clear from this Pull Request what file the SVG here is based on, in case we ever need that in the future.